corehq/apps/userreports/models.py

```diff
@@ -351,14 +351,22 @@ def get_es_index_settings(self):
         return {"settings": es_index_settings}
 
     def get_case_type_or_xmlns_filter(self):
+        """Returns a list of case types or xmlns from the filter of this data source.
+
+        If this can't figure out the case types or xmlns's that filter, then returns [None]
+        """
         def _get_property_value(config_filter, prop_name):
-            if (config_filter.get('type') != 'boolean_expression'
-                    or config_filter['operator'] != 'eq'):
-                return None
+            if config_filter.get('type') != 'boolean_expression':
+                return [None]
+
+            if config_filter['operator'] not in ('eq', 'in'):
+                return [None]
+
             expression = config_filter['expression']
             if expression['type'] == 'property_name' and expression['property_name'] == prop_name:
-                return config_filter['property_value']
-            return None
+                multiple = config_filter['operator'] == 'in'
+                return config_filter['property_value'] if multiple else config_filter['property_value']
```

Comment:
Both sides of this `if` are the same.

Reply:
https://github.com/dimagi/commcare-hq/pull/18460/commits/d580ddfff0d45ef615b924e43e98d7661006ecf9